CHANGELOG.md

```diff
@@ -29,6 +29,8 @@ The changelog for **SwifterSwift**. Also see the [releases](https://github.com/S
   - `removeFirst(where:)` array extension now is more generic `RangeReplaceableCollection` extensions. [#516](https://github.com/SwifterSwift/SwifterSwift/pull/516) by [LucianoPAlmeida](https://github.com/LucianoPAlmeida).
 - **RandomAccessCollection**:
   - `indices(of:)` array extension now is more generic `RandomAccessCollection` extensions. [#516](https://github.com/SwifterSwift/SwifterSwift/pull/516) by [LucianoPAlmeida](https://github.com/LucianoPAlmeida).
+- **UIView**:
+  - `fillToSuperview()` should be more performant. 
```

Comment:
Ah, post merge regret. We should clean up this CHANGELOG entry in another PR 😕 My bad guys

Reply:
Ooops, since I also approve my bad too :(( 
But it's simple I can fix it right :))